readme.md

```diff
@@ -35,12 +35,22 @@ erDiagram
     }
 
     user_details ||--|| users : user_id
+    user_details ||--|| files : avatar_file_id
```

Comment:
```suggestion
        user_details ||--o| files : avatar_file_id
```
As I can see from migartion avatar can be null. If this field can be nullable, then user can have 1 or 0 avatar (`-o|`).